src/joystream-lib/api.ts

```diff
@@ -286,6 +287,20 @@ export class JoystreamJs {
       protoMeta.setHasMarketing(inputMetadata.hasMarketing)
     }
 
+    const protoLicenseType = new License()
+    if (inputMetadata.license?.code != null) {
+      protoLicenseType.setCode(inputMetadata.license.code)
+    }
+    if (inputMetadata.license?.attribution != null) {
+      protoLicenseType.setAttribution(inputMetadata.license.attribution)
+    }
+    if (inputMetadata.license?.customText != null) {
+      protoLicenseType.setCustomText(inputMetadata.license.customText)
+    }
+    if (protoLicenseType != null) {
```

Comment:
I don't think we need that check, we create `protoLicenseType` ourselves so I don't think it will ever be nullish?